reactive/jenkins.py

```diff
@@ -177,11 +177,13 @@ def configure_plugins():
     status_set("maintenance", "Configuring plugins")
     remove_state("jenkins.configured.plugins")
     plugins = plugins_layer()
-    installed_plugins, incompatible_plugins = plugins.install(config("plugins"))
-    api = Api()
-    api.wait()  # Wait for the service to be fully up
+    plugins.backup()
+    try:
+        installed_plugins, incompatible_plugins = plugins.install(config("plugins"))
+        check_incompatible_plugins(incompatible_plugins)
+    except Exception:
```

Comment:
I tried being more specific when handling the exception but I couldn't find the right type of the exception. When I tried checking the exception that is raised I get this:
```
ConnectionError(MaxRetryError("HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /crumbIssuer/api/json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f285bcf3c18>: Failed to establish a new connection: [Errno 111] Connection refused',))",),)
```

I tried using `ConnectionError` and `MaxRetryError` but it didn't work.